### PR TITLE
docs(kiali): update development doc

### PR DESCRIPTION
**Pull request by josunect**
Adapt development documentation to alpha.

### CODE REVIEW

File: plugins/kiali/DEVELOPMENT.md [hunk 1]
```diff
@@ -112,14 +100,22 @@
    Add to locations in `app-config.local.yaml`
 
    ```yaml
+   backend:
+     auth:
+       dangerouslyDisableDefaultAuthPolicy: true
+   ---
    locations:
```
Comment (leandroberetta) [inline plugins/kiali/DEVELOPMENT.md:107]: This text goes below in the Add catalog section.

**Review by leandroberetta — CHANGES_REQUESTED** [review]

File: plugins/kiali/DEVELOPMENT.md (hunk 1 above)
Comment (josunect) [reply plugins/kiali/DEVELOPMENT.md]: Thanks! Updated

**Review by aljesusg — CHANGES_REQUESTED** [review]
We need to remove the point 3 about create the file packages/backend/src/plugins/kiali.tsx it's not required

**Review by aljesusg — APPROVED** [review]
LGFM

**Pull request merged**